src/material/schematics/ng-update/upgrade-rules/package-imports-v8/secondary-entry-points-rule.ts

```diff
@@ -22,6 +22,12 @@ const NO_IMPORT_NAMED_SYMBOLS_FAILURE_STR = `Imports from Angular Material shoul
  */
```

Comment:
Should the name of this file be changed to remove the `-v8`?

Reply:
I don't feel strongly either way